project/config/pacni/config/core.extension.yml

```diff
@@ -96,6 +97,12 @@ module:
   responsive_image: 0
   scheduled_transitions: 0
   search: 0
```

Comment:
You will need to disable the search module

Reply:
No worries, Ill put that through